docs/user-guide/fhir-server-config.md

```diff
@@ -0,0 +1,23 @@
+
+## Create a tenant for storing knowledge artifacts
+The measure evaluation engine relies on Measure and Library definitions that are assumed to be stored in an R4 compliant FHIR server. When using a multitenant IBM FHIR server installation, it is recommended that solutions consuming the engine configure a single, shared tenant in their FHIR server that will contain all knowledge artifacts. In our development environment, we call this tenant "knowledge". See the [IBM FHIR Server User's Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide/#322-tenant-specific-configuration-properties) for complete details on how to configure an additional tenant.
+
+## Enable searching for ValueSet resources vs. those defined in an IG
+In the fhir-server-config.json for the knowledge tenant created above, set fhirServer/core/serverRegistryResourceProviderEnabled to true to enable user-defined ValueSet resources to be searchable via the REST API. This is a required setting to enable search functionality in the cohort and measure authoring tool. See [the Dev Guide](/dev-guide/value-sets?id=fhir-server-setup) for complete details. The property is also described in the [IBM FHIR Server User's Guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide/).
+
+## Configure the FHIR Server's SSL certificate
+IBM FHIR comes preconfigured with only an SSL endpoint enabled. A clear text HTTP endpoint can be setup, but it is *strongly recommended* not to do this for any type of production environment, especially one that processes PHI data. 
+
+A default, well-known certificate is provided in the FHIR server installation with a subject name of localhost. The cohorting service will check the host name used to connect to the FHIR server against the certificate's subject and fail if there is no match. Users running the FHIR server on a different service from the cohorting engine code will need to update the FHIR server's SSL certificate and then make sure that the server's public key is trusted by the JVM that runs the cohorting component.
+
+When running IBM FHIR on IBM Cloud it is recommended to use a certificate generated by the Kubernetes certicates API. That process is described in the [Manage TLS Certificates in a Cluster](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/) section of the K8S user guide. Once you have a valid certificate request approved and issued in your cluster, you will need to configure  the IBM FHIR server to use that certificate and then configure the cohorting JVM to trust the server's public key. Both are typically done through Helm charts that are used to deploy the services to the IBM Cloud via a devops pipeline. Details are left up to the user, but internal teams may contact the cohorting component team for additional details on what is done within our team to meet IBM internal process guidelines.
```

Comment:
Spelling of certificates is incorrect.  "... generated by Kubernetes certificates API."